### ISSUE TITLE
[Designer]Bug 31601097: [Screen Reader - Adaptive Cards - FlightItinerary.JSON]: Narrator announces Image when focus jumps to the flight image in scan mode using up/down arrow key.

### ISSUE THREAD

**Issue by aaagrawal** [microsoft/AdaptiveCards#7632]
### Problem Description

[Bug 31601097](https://microsoft.visualstudio.com/OS/_workitems/edit/31601097): [Screen Reader - Adaptive Cards - FlightItinerary.JSON]: Narrator announces Image when focus jumps to the flight image in scan mode using up/down arrow key.

User Experience:

It will not impact the screen reader users if name property must will not defined for the Flight image.

Test Environment:

Application: Adaptive Cards
OS version: 2004 (OS Build 21301.1000)

Application Version: 2101.28.1.0

Tool Used: Accessibility Insights 

Pre-Requisite: Launch the Adaptive Card application.

Repro steps:

1. Launch the Adaptive Card application and Enable Narrator.

2. Adaptive Card UWP Visualizer screen starts appearing. 

3. Navigate to FlightItinerary.JSON button and activate it.

4. Adaptive card will be appeared on right pane.

4. Navigate to Flight image by using up and down arrow key and verify the issue.

Actual Result:

Name property is not defined for the Flight image. Narrator announces Image when focus jumps to the flight image in scan mode using up/down arrow key.

Expected result:

Narrator should announces information as Flight To' for the flight image when focus jumps to the flight image in scan mode using up/down arrow key.

### Card JSON

```JSON
Designer Bug. Repro on UWP Adaptive Card visualizer
```

**Comment by vagpt** [comment]
This issue is not repro hence closing this issue.

![Image](https://github.com/user-attachments/assets/9387c169-22c0-4a87-b60e-0e1d06e163c4)

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
